### ISSUE TITLE
Sensibo Sky - Multiple Air Conditions

### ISSUE THREAD

**Issue by Maddin3011** [ioBroker/AdapterRequests#542]
Sensibo Sky provides a Cloud based Bridge which can control a IR-based Air Condition from several Manufacturers.
(on/off; set timer; set temperature; current temperature; set mode).

https://sensibo.com
https://support.sensibo.com/l/en/category/yyjc33fcn7-api
https://support.sensibo.com/l/en/category/0q02y25s0o-integrations

**Comment by Maddin3011** [comment]
Maybe this could help to bring Sensibo to ioBroker:
https://github.com/nitaybz/homebridge-sensibo-ac

**Comment by gittjoel** [comment]
Sensibo air is also compatible with HomeKit now. 
Would be nice to find it in iobroker. THX

**Comment by MichaelSchaaf999** [comment]
We don't need HomeKit, but having Sensibo in IOBroker would be great.

**Comment by MichaelSchaaf999** [comment]
any progress here?

**Comment by MichaelSchaaf999** [comment]
Yes, tried, but as I am not really experienced with Homebridge (not used here for anything) it did not work. Sorry

**Comment by Alex0176** [comment]
there is another project where sensibo was integrated in a php solution:

https://github.com/1007/Symcon1007_Sensibo

**Comment by hytrek** [comment]
Hi there, I would also love to see an adapter to run my AC for sensibo!

**Comment by mcm1957** [comment]
Instead of me too üosts ülease click at thumbs- up at first (original) request
